README.md

```diff
@@ -216,6 +216,18 @@ In this tab there are the following settings:
 
 With these settings, you can add rich text content which will be shown on the imprint page.
 
+##### Contact
+
+With these settings, you can add rich text content which will be shown on the contact page.
+
+##### Help
+
+With these settings, you can add rich text content which will be shown on the help page.
+
+##### Maintenance
+
+With these settings, you can add rich text content which will be shown on the maintenance page.
```

Comment:
This may be confusing... are we referring to our static page with _information about maintenance_ or to the actual mainteance page generated with activated maintenance mode.